lib/ex_maps.ex

```diff
@@ -83,7 +83,7 @@ defmodule ExMaps do
           | transit_mode
           | transit_routing_preference
 
-  @type options :: [option: term]
+  @type options :: [{option, term}]
 
```

Comment:
Hi Susana, thanks for contribution! Just a quick question though - why would you like to change it? :)